programs/monaco_protocol/src/instructions/matching/matching_pool.rs

```diff
@@ -106,9 +103,7 @@ pub fn move_market_matching_pool_to_inplay(
         CoreError::MatchingMarketMatchingPoolAlreadyInplay
     );
 
-    market_matching_pool.move_to_inplay(&market.event_start_order_behaviour);
-
-    Ok(())
+    market_matching_pool.move_to_inplay(market_matching_queue, &market.event_start_order_behaviour)
```

Comment:
I think it is weird to be passing one object to another to check a condition - it creates a tangle between them. I thought that's what this file was supposed to contain. And I do  understand that we had some of that before so that's why this one is more of an opinion that request change.

Reply:
Yeah I agree. It was an attempt to centralise the condition and update code. I considered this function as the place for that but this is more like an instruction level function. Will refactor.